README.md

```diff
@@ -1 +1,122 @@
 # ChunkedBase.jl
+
+The package handles the ingestion of data chunks and the distribution & synchronization of work that happens on these chunks in parallel. It came into existence while refactoring the `ChunkedCSV.jl` and `ChunkedJSON.jl` packages and was designed to be extended by packages like these. It is a package used to write parser packages.
```

Comment:
And can you make ChunkedCSV and ChunkedJSON into links too?
I actually couldn't find ChunkedJSON.jl on google... Is it ChunkedJSONLines.jl?

Also also, can we move these into an org? I thought that was one of the goals of the Q2 project, was that we didn't want to have RAICode depending on any packages in personal repos. I know that you guys were planning to merge into CSV.jl and then gave up on that, but I think that this is still an important goal. Can we squeeze that into this month?

Reply:
The packages live in the RAICode organization but are not registered.
https://github.com/RelationalAI/ChunkedCSV.jl
https://github.com/RelationalAI/ChunkedJSONL.jl

Reply:
added this to the Epic for this month

Reply:
Aha, thanks. I'd been looking at the `drvi` version; i didn't realize it was forked, not moved.